demos/Export.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Export | Keyguard Demo</title>
+</head>
+<body>
+<link href="../src/nimiq-style.css" rel="stylesheet">
+<script src="../node_modules/@nimiq/core-web/web-offline.js"></script>
+<script src="../src/lib/Constants.js"></script>
+<script src="../src/lib/Key.js"></script>
+<script src="../src/lib/KeyStore.js"></script>
+<script src="../node_modules/@nimiq/rpc/dist/rpc.umd.js"></script>
+
+<div>
+    <button class="bip39">Export Words (BIP39)</button>
```

Comment:
```suggestion
    <button class="bip39">Export (BIP39)</button>
```
following lines the same.